app/controllers/concerns/idv/acuant_concern.rb

```diff
@@ -0,0 +1,39 @@
+module Idv
+  module AcuantConcern
```

Comment:
This concern shows how I'm thinking code sharing can work between hybrid and non-hybrid flows--it depends on the controller itself to implement `document_capture_session_uuid`, and does not contain references to e.g. `flow_session`